buttons/src/test/kotlin/com/decathlon/vitamin/compose/buttons/VitaminIconButtonsPrimaryTest.kt

```diff
@@ -0,0 +1,105 @@
+package com.decathlon.vitamin.compose.buttons
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import app.cash.paparazzi.Paparazzi
+import com.decathlon.vitamin.compose.buttons.utils.ButtonVariantsFactory
+import com.decathlon.vitamin.compose.buttons.utils.Theme
+import com.decathlon.vitamin.compose.buttons.utils.Variant
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+import com.google.testing.junit.testparameterinjector.TestParameter
+import com.google.testing.junit.testparameterinjector.TestParameterInjector
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+@RunWith(TestParameterInjector::class)
+class VitaminIconButtonsPrimaryTest(
```

Comment:
Where are screenshots with icon buttons? Can't find them in the PR.

Reply:
There are in the PR, for example : [The snapshot of the conversion icon alone button](https://github.com/Decathlon/vitamin-compose/blob/09cfeae71faae813bab4183ef59c64e5cc5e9c5d/buttons/src/test/snapshots/images/com.decathlon.vitamin.compose.buttons_VitaminIconButtonsPrimaryTest_medium%5BConversion%2CLight%5D.png)
But this is strange in the modified files list the image preview is not loading, I don't know why.